.github/workflows/tests.yml

```diff
@@ -36,16 +36,10 @@ jobs:
           echo 'GYP_MSVS_OVERRIDE_PATH=C:\\Dummy' >> $Env:GITHUB_ENV
       - name: Lint Python
         if: matrix.os == 'ubuntu-latest'
-        run: |
-          # stop the build if there are Python syntax errors or undefined names
-          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        run: flake8 . --ignore=E203,W503  --max-complexity=101 --max-line-length=88 --show-source --statistics
```

Comment:
Should we keep max-line-length to 127 ?

Reply:
88 is the way to go... https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length